paymentproxy/proxy.go

```diff
@@ -126,7 +148,7 @@ func (p *PaymentProxy) handleDestinationResponse(r *http.Response) error {
 	// s.Delta is amount our balance increases by adding this voucher
 	// AKA the payment amount we received in the request for this file
 	if cost > s.Delta.Uint64() {
-		return createPaymentError(fmt.Errorf("payment of %d required, the voucher only resulted in a payment of %d", cost, s.Delta.Uint64()))
+		return createPaymentError(fmt.Errorf("payment of %d attoFIL required, the voucher only resulted in a payment of %d attoFIL", cost, s.Delta.Uint64()))
```

Comment:
We would prefer to not hardcode this here. However, the entire demo is already quite tightly coupled to Filecoin, so it is not high priority to fix.